packages/stacks-docs/product/components/vote.html

```diff
@@ -0,0 +1,242 @@
+                        <span class="s-vote--votes">
+                            <span class="s-vote--total">{{ vote.count }}</span>
+                        </span>
+                        <button class="s-vote--btn">
+                            {% icon "Vote16Down" %}
+                            <span class="v-visible-sr">downvote</span>
+                        </button>
+                    </div>
+                </div>
+            {% endfor %}
+        </div>
+    </div>
+
+    {% header "h3", "Expanded" %}
+    <p class="stacks-copy">
+        Include the <code class="stacks-code">.s-vote__expanded</code> modifier to show upvote and downvote counts in addition to the total vote count. This modifier hides <code class="stacks-code">.s-vote--total</code> and shows <code class="stacks-code">.s-vote--upvotes</code> and <code class="stacks-code">.s-vote--downvotes</code> instead.
+    </p>
+
+    <div class="stacks-preview mb24">
+{% highlight html %}
+<div class="s-vote">
+    <button class="s-vote--btn">
+        @Svg.Vote16Up
+        <span class="v-visible-sr">upvote</span>
+    </button>
+    <button class="s-vote--votes">
+        <span class="s-vote--upvotes">12</span>
+        <span class="s-vote--total">20</span>
+        <span class="s-vote--downvotes">8</span>
+    </button>
+    <button class="s-vote--btn">
+        @Svg.Vote16Down
+        <span class="v-visible-sr">downvote</span>
+    </button>
+</div>
+{% endhighlight %}
+        <div class="stacks-preview--example d-flex ai-start fw-wrap g64">
+            {% for vote in vote.groups.expanded %}
+                <div class="d-flex fd-column ai-center g16">
+                    <div class="s-vote s-vote__expanded">
+                        <button class="s-vote--btn">
+                            {% icon "Vote16Up" %}
+                            <span class="v-visible-sr">upvote</span>
+                        </button>
+                        <span class="s-vote--votes">
+                            <span class="s-vote--upvotes">{{ vote.positive }}</span>
+                            <span class="s-vote--total">{{ vote.count }}</span>
+                            <span class="s-vote--downvotes">{{ vote.negative }}</span>
+                        </span>
+                        <button class="s-vote--btn">
+                            {% icon "Vote16Down" %}
+                            <span class="v-visible-sr">downvote</span>
+                        </button>
+                    </div>
+                </div>
+            {% endfor %}
+        </div>
+    </div>
+
+    {% header "h3", "Horizontal" %}
+    <p class="stacks-copy">
+        Apply the <code class="stacks-code">.s-vote__horizontal</code> modifier to arrange the vote buttons and counts in a horizontal layout. This layout does not officially support expanded vote count. This configuration is best suited scenarios such as comment voting, where a more compact design is preferred.
```

Comment:
small nit: missing "for" 
```suggestion
        Apply the <code class="stacks-code">.s-vote__horizontal</code> modifier to arrange the vote buttons and counts in a horizontal layout. This layout does not officially support expanded vote count. This configuration is best suited for scenarios such as comment voting, where a more compact design is preferred.
```